server/jamovi/common/column.cpp

```diff
@@ -21,7 +22,8 @@ Column::Column(DataSet *parent, MemoryMap *mm, ColumnStruct *rel)
     _rel = rel;
 }
 
-int Column::id() const {
+int Column::id() const
+{
     return struc()->id;
```

Comment:
let's not commit style changes either ... if we're going to change the style (for existing code) then we'll do it in a separate commit too ... i.e. we try and keep commits very focused.

Reply:
yep. was an accident. will fix